sanic/blueprint_group.py

```diff
@@ -187,35 +187,14 @@ def __len__(self) -> int:
         """
         return len(self._blueprints)
 
-    def _sanitize_blueprint(self, bp: "sanic.Blueprint") -> "sanic.Blueprint":
-        """
-        Sanitize the Blueprint Entity to override the Version and strict slash
-        behaviors as required.
-
-        :param bp: Sanic Blueprint entity Object
-        :return: Modified Blueprint
-        """
-        if self._url_prefix:
-            merged_prefix = "/".join(
-                u.strip("/") for u in [self._url_prefix, bp.url_prefix or ""]
-            ).rstrip("/")
-            bp.url_prefix = f"/{merged_prefix}"
-        for _attr in ["version", "strict_slashes"]:
-            if getattr(bp, _attr) is None:
-                setattr(bp, _attr, getattr(self, _attr))
-        if bp.version_prefix == "/v":
-            bp.version_prefix = self._version_prefix
-
-        return bp
-
     def append(self, value: "sanic.Blueprint") -> None:
```

Comment:
Can't we use [PEP 563](https://www.python.org/dev/peps/pep-0563/#enabling-the-future-behavior-in-python-3-7) style annotations, since Sanic is Python 3.7+?